### PR TITLE
Add top bar to GrowPod

### PR DESCRIPTION
**Pull request by codyjrivera**
Adds a top app bar to Growpod as well as check to see if the user is logged in (right now, the user is always logged in). This is based on what is currently in Master, so the new theming has not been applied.

image

### CODE REVIEW

File: src/main/webapp/src/app/app.component.html [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
-      </span>
-    </a>
-  </mat-toolbar-row>
-</mat-toolbar>
-
-<!-- Routed page contents -->
-<router-outlet></router-outlet>
+<!-- Displays app contents if user is logged in, otherwise directs user to a home/login page -->
+<div *ngIf="loggedIn; else homePage">
+  <mat-toolbar id="growpod-top-bar" color="primary">
```
Comment (stephypy) [inline src/main/webapp/src/app/app.component.html:3]: We won't be using the id growpod-top-bar so this can be removed

Comment (codyjrivera) [reply src/main/webapp/src/app/app.component.html]: Addressed.

File: src/main/webapp/src/app/common/growpod-page-styles.css [hunk 2]
```diff
@@ -9,3 +9,21 @@
 .text-center {
   text-align: center;
 }
+
+.profile-pic-container-s {
+  position: relative;
+  width: 33%; /* The size you want */
+  margin-left: auto;
```
Comment (caroqliu) [inline src/main/webapp/src/app/common/growpod-page-styles.css:16]: `margin-left` and `margin-right` can be consolidated if they are the same value

File: src/main/webapp/src/app/common/growpod-page-styles.css [hunk 3]
```diff
@@ -9,3 +9,21 @@
...
+  margin-right: auto;
+  background-color: #fff;
+  border: 4px solid black;
+  border-radius: 50%;
+  -moz-border-radius: 50%;
+  -webkit-border-radius: 50%;
+}
+
+.profile-pic-container-s:after {
+  content: '';
```
Comment (caroqliu) [inline src/main/webapp/src/app/common/growpod-page-styles.css:26]: What is the intention with this line?

Comment (codyjrivera) [reply src/main/webapp/src/app/common/growpod-page-styles.css:26]: Without this line, the circle's height is not the same as its width.

File: src/main/webapp/src/app/common/growpod-page-styles.css (hunk 2 above)
Comment (codyjrivera) [reply src/main/webapp/src/app/common/growpod-page-styles.css:16]: Will change to `margin: 0px auto`.

File: src/main/webapp/src/app/app.component.html [hunk 4]
```diff
@@ -1,21 +1,43 @@
...
+        <h4 class="text-center">David Ladd</h4>
+        <a routerLink="/page/user-profile">
+          <button mat-menu-item>
+            <mat-icon>assignment_ind</mat-icon>
+            <span>My Profile</span>
+          </button>
+        </a>
+        <button mat-menu-item disabled>
+          <mat-icon>login</mat-icon>
+          <!-- https://imgur.com/fub9WUs -->
```
[the remark this answers is not in the document]
Comment (codyjrivera) [reply src/main/webapp/src/app/app.component.html:31]: It's a Prequel meme joke based on the fact that I am using Material's log in icon to depict logging out. I will remove it to match the rest of the file.

File: src/main/webapp/src/app/common/growpod-page-styles.css (hunk 2 above)
Comment (samouri) [reply src/main/webapp/src/app/common/growpod-page-styles.css]: nit: `0px;` doesn't actually need the unit. can write `0;`

File: src/main/webapp/src/app/app.component.html [hunk 5]
```diff
@@ -1,21 +1,42 @@
...
+        </button>
+      </mat-menu>
+    </mat-toolbar-row>
+  </mat-toolbar>
+  <!-- Routed page contents -->
+  <router-outlet></router-outlet>
+</div>
+<!-- Error homepage -->
+<ng-template #homePage>
+  <h1>You aren't signed into GrowPod )-;</h1>
```
Comment (samouri) [inline src/main/webapp/src/app/app.component.html:41]: (˚Õ˚)ر ~~~~╚╩╩╝

File: src/main/webapp/src/app/common/growpod-page-styles.css (hunk 2 above)
Comment (codyjrivera) [reply src/main/webapp/src/app/common/growpod-page-styles.css]: Done.

**Pull request merged**